### PR TITLE
TaggedEventSeries events do not need to be split.

### PR DESCRIPTION
**Pull request by tasomaniac**
Only the events in the main page namely `GdgEventListFragment` should be split.

### CODE REVIEW

File: app/src/main/java/org/gdg/frisbee/android/eventseries/GdgEventListFragment.java [hunk 1]
```diff
@@ -96,6 +100,45 @@ private void onNotFound() {
...
+     * Helper method that will split events into upcoming / past
+     * and add to the adapter of this list fragment
+     *
+     * @param events
+     *      The events list to be added
+     */
+    private void splitEventsAndAddToAdapter(Collection<? extends SimpleEvent> events) {
+        Pair<List<SimpleEvent>, List<SimpleEvent>> pair = splitEventsList(events);
+        mAdapter.addAll(pair.first);
+        mAdapter.addAll(pair.second);
```
Comment (stefanhoth) [inline app/src/main/java/org/gdg/frisbee/android/eventseries/GdgEventListFragment.java]: I'm a bit confused. Is that splitting basically a way of sorting or are they used somewhere separately?

**Comment by stefanhoth** [comment]
I'm obviously not too familiar with the code base so excuse me. But what are tagged events and why aren't they split?

File: app/src/main/java/org/gdg/frisbee/android/eventseries/GdgEventListFragment.java (hunk 1 above)
Comment (tasomaniac) [inline app/src/main/java/org/gdg/frisbee/android/eventseries/GdgEventListFragment.java]: We actually have sorting in the tagged events pages like devfest 's. Splitting is in the main page where we split past and upcoming events of A chapter.

Comment (stefanhoth) [inline app/src/main/java/org/gdg/frisbee/android/eventseries/GdgEventListFragment.java]: But first it's split and that added to the same adapter without any distinction. I'm just wondering of the same could be achieved by a comparator.

Comment (tasomaniac) [inline app/src/main/java/org/gdg/frisbee/android/eventseries/GdgEventListFragment.java]: Yeah exactly. That's what I said in the original PR when this is first
implemented. Splitting means here that showing the upcoming items first. It
would be better to do like that but there are too many things to do for me.

**Comment by stefanhoth** [comment]
OK, LGTM. Thanks for the explanations.

**Comment by tasomaniac** [comment]
Thank you for reviewing it :)

**Pull request merged**